addons/mail/views/discuss_public_templates.xml

```diff
@@ -1,14 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <template id="mail.discuss_public_layout">&lt;!DOCTYPE html&gt;
+    <template id="mail.discuss_public_template" name="Discuss Public Template">&lt;!DOCTYPE html&gt;
```

Comment:
I would avoid these unnecessary renames (will require migration script).

Reply:
revert naming (only for the template id)